shell-vars.wake

```diff
@@ -0,0 +1,22 @@
+# Allow the user to have a shell environment variable to change
+# environmental setting within their wake rules.
+# Examples would include the publishing/setting of locations of toolchains.
+# For example:
+#
+#   $ export WAKE_ENV=home
+#   $ wake -x 'getFooLocation'
+#     /home/user/bin/foo
+#   $ export WAKE_ENV=work
+#   $ wake -x 'getFooLocation'
+#     /opt/foo
```

Comment:
I don't understand these examples, how does `home` give you `/home/user/bin/foo` which `work` gives you `/opt/foo`?

Reply:
Ah I see, got to keep reading, you basically have publishes that are predicated on the value of `WAKE_ENV`